src/test/java/org/isf/pricesothers/test/TestPricesOthersContext.java

```diff
@@ -35,8 +35,8 @@ public void deleteNews(
     {
 		jpa.beginTransaction();			
 		jpa.createQuery("SELECT * FROM PRICESOTHERS", PricesOthers.class, false);
-		List<PricesOthers> PricesOtherss = (List<PricesOthers>)jpa.getList();
-		for (PricesOthers pricesOthers: PricesOtherss) 
+		List<PricesOthers> PricesOthers = (List<PricesOthers>)jpa.getList();
+		for (PricesOthers pricesOthers: PricesOthers)
```

Comment:
according to standard Java naming this should also have the 'P' lowercase?

Reply:
Good catch; this is a result of an over zealous change and replace action.   Fixed in another [PR](https://github.com/informatici/openhospital-core/pull/115).